src/js/actions/layers.js

```diff
@@ -662,7 +656,7 @@ define(function (require, exports) {
         if (linkedLayers.isEmpty()) {
             return Promise.resolve();
         }
-        return this.transfer(resetBounds, document, linkedLayers, true);
+        return this.transfer(resetBounds, document, linkedLayers, true, true);
```

Comment:
for point, but to clarify: these will soon be completely refactored out.  ResetBounds will no longer have these arguments at all.